.github/workflows/go.yml

```diff
@@ -0,0 +1,88 @@
+name: CI
+
+on:
+  schedule:
+    # run every Sunday to re-populate caches after they are cleaned on Saturday
```

Comment:
That doesn't really work with the recent version of the cache action; probably should be removed everywhere